src/parser.py

```diff
@@ -60,6 +60,7 @@ def parse(self, path:str):
         if subdirs["done"]:
```

Comment:
The directory name should probably not be hard-coded. I suggest we change it in a future commit

Reply:
agreed, probably could be defined in settings as a global variable

Reply:
It already is, see  `common/settings.py`   :-)